<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>495</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HAP Convert</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QPushButton" name="bt_open">
      <property name="font">
       <font>
        <family>08서울남산체 B</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Open</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="le_filepath">
      <property name="font">
       <font>
        <family>08서울남산체 B</family>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QTextEdit" name="txt_fileinfo">
      <property name="font">
       <font>
        <family>08서울남산체 B</family>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QGroupBox" name="groupBox">
      <property name="font">
       <font>
        <family>08서울남산체 B</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="title">
       <string>설정</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="3">
        <widget class="QLabel" name="lb_height">
         <property name="font">
          <font>
           <family>08서울남산체 B</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>세로해상도</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="le_width">
         <property name="font">
          <font>
           <family>08서울남산체 B</family>
           <pointsize>10</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item row="1" column="4">
        <widget class="QComboBox" name="cbb_option">
         <property name="font">
          <font>
           <family>08서울남산체 B</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="placeholderText">
          <string/>
         </property>
         <property name="modelColumn">
          <number>0</number>
         </property>
         <item>
          <property name="text">
           <string>hap</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>hap_alpha</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>hap_q</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <family>08서울남산체 B</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>변환옵션</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QCheckBox" name="cb_originalRes">
         <property name="font">
          <font>
           <family>08서울남산체 B</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>원본해상도</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="lb_width">
         <property name="font">
          <font>
           <family>08서울남산체 B</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>가로해상도</string>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QLineEdit" name="le_height">
         <property name="font">
          <font>
           <family>08서울남산체 B</family>
           <pointsize>10</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="lb_giho">
         <property name="text">
          <string>x</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="4" column="0" colspan="2">
     <widget class="QProgressBar" name="progressBar">
      <property name="font">
       <font>
        <family>08서울남산체 B</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QPushButton" name="bt_convert">
      <property name="font">
       <font>
        <family>08서울남산체 B</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>변환</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>520</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>bt_open</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>fileOpen()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>43</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel">
     <x>36</x>
     <y>58</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bt_convert</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>fileConvert()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>432</x>
     <y>429</y>
    </hint>
    <hint type="destinationlabel">
     <x>441</x>
     <y>275</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cb_originalRes</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>getCheck()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>57</x>
     <y>392</y>
    </hint>
    <hint type="destinationlabel">
     <x>7</x>
     <y>385</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbb_option</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>getOpt()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>396</x>
     <y>390</y>
    </hint>
    <hint type="destinationlabel">
     <x>514</x>
     <y>389</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>fileOpen()</slot>
  <slot>fileConvert()</slot>
  <slot>getCheck()</slot>
  <slot>getOpt()</slot>
 </slots>
</ui>
